calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/camera_calibrator.py

```diff
@@ -651,15 +612,115 @@ def start(
+        self.consumed_data_signal.connect(self.on_consumed)
+
+        self.central_widget = QWidget(self)
+        self.setCentralWidget(self.central_widget)
+        self.layout = QHBoxLayout(self.central_widget)
+
+        # Image View
+        self.make_image_view()
+
+        # Menu Widgets
+        self.left_menu_widget = QWidget(self.central_widget)
+        self.left_menu_widget.setFixedWidth(300)
+        self.left_menu_layout = QVBoxLayout(self.left_menu_widget)
+        self.left_menu_layout.setAlignment(Qt.AlignTop)
+
+        self.right_menu_widget = QWidget(self.central_widget)
+        self.right_menu_widget.setFixedWidth(300)
+        self.right_menu_layout = QVBoxLayout(self.right_menu_widget)
+        self.right_menu_layout.setAlignment(Qt.AlignTop)
+        self.setEnabled(False)
+
+        # Creating the UI elements after selecting CALIBRATION or EVALUATION
+        # Mode group
+        self.make_mode_group()
+
+        # Create solver selector
+        self.make_solver_group()
+
+        # Calibration group
+        if self.operation_mode == OperationMode.CALIBRATION:
+            self.make_calibration_group()
+
+        # Detector group
+        self.make_detector_group()
+
+        # Detections group
+        self.make_detection_group()
+
+        # Data collection group
+        if self.operation_mode == OperationMode.CALIBRATION:
+            self.make_data_collection_group()
+
+        # Visualization group
+        self.make_visualization_group()
+
+        # self.menu_layout.addWidget(label)
+        self.left_menu_layout.addWidget(self.solver_group)
+        if self.operation_mode == OperationMode.CALIBRATION:
+            self.left_menu_layout.addWidget(self.calibration_group)
+        self.left_menu_layout.addWidget(self.detector_options_group)
+        self.left_menu_layout.addWidget(self.raw_detection_results_group)
+        self.left_menu_layout.addWidget(self.single_shot_detection_results_group)
+
+        self.right_menu_layout.addWidget(self.mode_options_group)
+        if self.operation_mode == OperationMode.CALIBRATION:
+            self.right_menu_layout.addWidget(self.data_collection_group)
+        self.right_menu_layout.addWidget(self.visualization_options_group)
+
+        self.layout.addWidget(self.graphics_view)
+
+        self.layout.addWidget(self.left_menu_widget)
+        self.layout.addWidget(self.right_menu_widget)
+        self.show()
         self.setEnabled(True)
 
-        self.setWindowTitle(f"Camera intrinsics calibrator ({self.data_source.get_camera_name()})")
+        if self.operation_mode == OperationMode.CALIBRATION:
+            self.setWindowTitle(
+                f"Camera intrinsics calibrator ({self.data_source.get_camera_name()})"
+            )
+        if self.operation_mode == OperationMode.EVALUATION:
+            self.setWindowTitle(
+                f"Camera intrinsics Evaluation Mode ({self.data_source.get_camera_name()})"
```

Comment:
let's try not to mix cases

Reply:
I'm not sure what to do here

Reply:
`Camera intrinsics Evaluation Mode` to `Camera intrinsics evaluation mode`

Reply:
f2ad5cce4cffc72284d0b575020c40851475872e